recipes/flac/all/conanfile.py

```diff
@@ -71,3 +71,5 @@ def package_info(self):
         self.cpp_info.libs = ['FLAC++', 'FLAC']
         if not self.options.shared:
             self.cpp_info.defines = ["FLAC__NO_DLL"]
+            if self.settings.os == "Linux":
+                self.cpp_info.system_libs += ["m"]
```

Comment:
Consider using
```
self.cpp_info.libs.append("m")
```

Reply:
I think the following should have the same effect in python:
```
self.cpp_info.libs.append("m")
self.cpp_info.libs.extend(["m"])
self.cpp_info.libs += ["m"]
self.cpp_info.libs = self.cpp_info.libs + ["m"]
```
just a matter of styling, for me everything above is equally easy to understand

Reply:
True that, but we should have a consistent style. That's what most of the hooks enforce right?
We have a hook for "new line at the end of file". What is that, if not style? :)

Reply:
new line at the end of file is not a style. that's C standard, which requires each source file to have a end of line as very last character. not having EOL at the end of .c file, emits a compiler warning, and this might cause build failures compiling with **-Werror** or **/WX**.
for the python, the all four variants are valid and emit no linter warnings AFAIK.
do you know if pep has recommendation on particular syntax?
I believe, currently different recipes are using different syntax, and none of them shows strong advantages over others.

Reply:
> new line at the end of file is not a style. that's C standard, which requires each source file to have a end of line as very last character. not having EOL at the end of .c file, emits a compiler warning, and this might cause build failures compiling with **-Werror** or **/WX**.

That is true, but the hook checks the yml files, too ;)

> for the python, the all four variants are valid and emit no linter warnings AFAIK.

True, but it is against the Zen of Python. This is perl-ish :)

> do you know if pep has recommendation on particular syntax?
> I believe, currently different recipes are using different syntax, and none of them shows strong advantages over others.

```
~/tools/conan-center-index$ ag self.cpp_info.libs | grep -c append
37
~/tools/conan-center-index$ ag self.cpp_info.libs | grep -c extend
9
~/tools/conan-center-index$ ag self.cpp_info.libs | grep -c "+="
2
~/tools/conan-center-index$ ag self.cpp_info.libs | grep -c " \+ \["
0
```

I see extend used with another list and append used with single element.